src/test/java/org/opendevstack/provision/controller/ProjectApiControllerTest.java

```diff
@@ -249,7 +244,7 @@ public void addProjectWithOC() throws Exception {
                     .andExpect(MockMvcResultMatchers.status().isOk())
                     .andDo(MockMvcResultHandlers.print());
 
-                verifyAddProjectAdapterCalls(times(1), times(confluenceAdapterEnabled ? 1 : 0));
```

Comment:
This line is actually ok. I realized that the method `verifyAddProjectAdapterCalls` has a bug. 
To fix it, the folllowing code needs to be added to the method otherwise it doesn't make sense to pass the 2nd variable:
```
 Mockito.verify(confluenceAdapter, confluenceTimes)
            .createCollaborationSpaceForODSProject(isNotNull());
```

Reply:
done